src/app/system/account-number-preferences/account-number-preferences.component.scss

```
@@ -0,0 +1,7 @@
+table {
+    width: 100%;
+
+    .select-row:hover {
+      cursor: pointer;
+    }
+  }
```

```suggestion
}
```
Add a line at the end of file.